earth_enterprise/docs/developer/BuildSystem.md

```diff
@@ -0,0 +1,130 @@
+# Overview of the Build System Used in Open GEE
+
+There are, at least, three separate software products in the Open GEE
```

Comment:
Commas look to be overkill

Reply:
I'm following the grammar for parenthetical expressions.  These are separated by commas, dashes, or brackets from the rest of the sentence, and commas are the most usual for this expression.

Reply:
Writing it this way doesn't look right.  If it conforms with the style used elsewhere in documentation, then it is ok I guess.